mcstatus/server.py

```diff
@@ -24,15 +25,18 @@
 __all__ = ["JavaServer", "BedrockServer", "MinecraftServer", "MinecraftBedrockServer"]
 
 
-class JavaServer:
-    """Base class for a Minecraft Java Edition server.
+class MCServer(ABC):
+    """Base abstract class for a general minecraft server.
 
-    :param str host: The host/address/ip of the Minecraft server.
+    This class only contains the basic logic shared across both java and bedrock versions,
+    it doesn't include any version specific settings and it can't be used to make any requests.
+
+    :param str host: The host/ip of the minecraft server.
     :param int port: The port that the server is on.
-    :param float timeout: The timeout in seconds before failing to connect.
+    :param float timeout: Timeout, in seconds, before failing to connect.
```

Comment:
Correcting the punctuation here, but not in other places that have the same sentence?